e2e/cypress/integration/channel/message_parse_spec.js

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// [number] indicates a test step (e.g. # Go to a page)
+// [*] indicates an assertion (e.g. * Check the title)
+// Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Message', () => {
```

Comment:
Not mentioned in the ticket (as this is old generated help-wanted) but please change to
```suggestion
describe('Messaging', () => {
```